docs/site/Limit-filter.md

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Limit filter'
+lang: en
+keywords: LoopBack 4.0, LoopBack 4, limit
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Limit-filter.html
+summary: A <i>limit</i> filter specifies a set of logical conditions to match, similar to a LIMIT clause in a SQL query.
+---
+
+A _limit_ filter limits the maximum number of records it returns.
+
+{% include content/angular-methods-caveat.html lang=page.lang %}
+
+### REST API
+
+<pre>
+filter[limit]=<i>n</i>
+</pre>
+
+You can also use [stringified JSON format](Querying-data.html#using-stringified-json-in-rest-queries) in a REST query.
```

Comment:
Let's show a example of URL with stringified JSON format for the `limit` filter.